tests-semanticdb/src/test/scala/scala/meta/tests/Utils.scala

```diff
@@ -1,16 +1,17 @@
 package scala.meta.tests
 
-import scala.meta.io.AbsolutePath
-
+import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.Paths
-
 import scala.annotation.tailrec
+import scala.meta.io.AbsolutePath
 
 object Utils {
 
-  def getResourceOpt(file: String): Option[Path] = Option(getClass.getResource("/" + file))
-    .map(x => Paths.get(x.toURI))
+  def getResourceOpt(file: String): Option[Path] = {
+    val path = Paths.get(BuildInfo.resourcesDirectory).resolve(file)
```

Comment:
Not sure when this stopped working, but I can't make it work locally. This wa we actually have the real disk path.